Examples/01_Optimization/01_Water/run.sh

```diff
@@ -1,7 +1,7 @@
 cp ../../../qxmd .
-cp /content/QXMD-CYBER-MAGICS-2023JUNE/util/qxREAD.py /content/QXMD-CYBER-MAGICS-2023JUNE/Examples/01_Optimization/01_Water/analysis/DOS/.
-cp /content/QXMD-CYBER-MAGICS-2023JUNE/util/qxREAD.py /content/QXMD-CYBER-MAGICS-2023JUNE/Examples/01_Optimization/01_Water/analysis/eng/.
-cp /content/QXMD-CYBER-MAGICS-2023JUNE/util/qxREAD.py /content/QXMD-CYBER-MAGICS-2023JUNE/Examples/01_Optimization/01_Water/analysis/pdb/.
+cp /content/QXMD/util/qxREAD.py /content/QXMD/Examples/01_Optimization/01_Water/analysis/DOS/.
```

Comment:
`/content/` is still hard coding the paths which could break things in the future.

Can you try relative paths by adding a cell in Google collab to define an environment variable `SOURCE_DIR` such that this becomes `${SOURCE_DIR}/util/qxREAD.py`